src/components/NavigationGroup.svelte

```diff
@@ -1,6 +1,6 @@
 {{#if separator == 'top'}}<hr />{{/if}}
 {{#if group.length}}
-<ul class='coz-nav-group'>
+<ul class='{{`coz-nav-group${wrappingClass}`}}'>
```

Comment:
No there is a space (https://github.com/cozy/cozy-bar/pull/44/files#diff-b97a1bf8e7abb24f9a5b3e2a65fcdf69R21) but I think I can put that space here for better understanding 👍